packages/example-app-base/src/components/ComponentPage/ComponentPage.tsx

```diff
@@ -11,23 +11,42 @@ export interface IComponentPageSection {
 }
 
 export interface IComponentPageProps {
+  /** Component Title **/
   title: string;
+  /** Component Name **/
   componentName: string;
+  /** Component examples **/
   exampleCards?: JSX.Element;
+  /** Array of implementation examples, displayed in the order defined */
   implementationExampleCards?: JSX.Element;
+  /** Component properties table(s) **/
   propertiesTables?: JSX.Element;
+  /** Component best practices **/
   bestPractices?: JSX.Element;
+  /** Component dos **/
   dos?: JSX.Element;
+  /** Component donts **/
   donts?: JSX.Element;
+  /** Component overview **/
   overview?: JSX.Element;
+  /** Related link */
   related?: JSX.Element;
+  /** Header visibility flag */
   isHeaderVisible?: boolean;
+  /** Badges visibility flag **/
   areBadgesVisible?: boolean;
+  /** className of the component being documented */
   className?: string;
+  /** Status of the component; e.g. keyboard accessible */
   componentStatus?: JSX.Element;
+  /** Pass through other sections for ComponentPage */
   otherSections?: IComponentPageSection[];
+  /** Allows native props */
   allowNativeProps?: boolean | string;
+  /** Native props root element */
   nativePropsElement?: string | string[] | undefined;
+  /** Includes the feedback section **/
+  feedback?: boolean;
```

Comment:
call this `isFeedbackVisible`?  (Consistency with other things in the props here.